### PR TITLE
Handle 2D input for BatchNormalization

### PR DESCRIPTION
**Pull request by jiafatom**

### CODE REVIEW

File: keras2onnx/ke2onnx/batch_norm.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 def convert_keras_batch_normalization(scope, operator, container):
     op = operator.raw_operator
-    if op.axis != 3 and op.axis != -1:
+    if (op.axis != 3 and op.axis != -1) or len(op.input_shape) == 2:
         adjusted_input_name = operator.inputs[0].full_name
```
Comment (vinitra) [inline keras2onnx/ke2onnx/batch_norm.py]: Is the op.axis portion always false in the 2D case? The "or" might not let it reach the second part of the if staement.

Comment (jiafatom) [reply keras2onnx/ke2onnx/batch_norm.py]: yes, for 2d case, dimension 0 is batch, so the data is actually dimension 1. So there is no notion of channel at this point. So we should not apply transpose.
